content/millennium/r4/documents/document-reference.md

```diff
@@ -109,9 +109,65 @@ List an individual DocumentReference by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Create
+
+Create a new document reference.
+
+    POST /DocumentReference
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* All provided dates must have a time component.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:document_reference, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr-code.cerner.com/r4/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/DocumentReference
+
+### Body
+
+<%= json(:R4_DOCUMENT_REFERENCE_POST) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Fri, 14 Feb 2020 22:05:40 GMT
+Etag: W/"12793861"
+Location: https://fhir-ehr-code.cerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/DocumentReference/16885181
```

Comment:
tenant key needs to be updated here as well

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/c11917e277b1bde8d54f91f63e4134060da17f1e